packages/@aws-cdk/core/lib/resource.ts

```diff
@@ -14,6 +14,28 @@ export interface IResource extends IConstruct {
    * The stack in which this resource is defined.
    */
   readonly stack: Stack;
+
+  /**
+   * The AWS account ID that this resource belongs to.
+   * For resources that are created and managed by the CDK
+   * (generally, those created by creating new class instances like Role, Bucket, etc.)
+   * this is always the same as the account of the stack they belong to;
```

Comment:
So it’s not “always the same”. Phrasing is a bit awkward

Reply:
For the created ones, it is always the same.

But of course, if you have a better wording, I'm all ears!